geojson/geometry.py

```diff
@@ -53,6 +53,9 @@ def __init__(self, geometries=None, **extra):
         super(GeometryCollection, self).__init__(**extra)
         self["geometries"] = geometries or []
 
+    def errors(self):
+        return any(list(geom.errors() for geom in self['geometries']))
```

Comment:
I think `errors` is supposed to return a list of errors instead of a boolean, like the other implementations